gulpfile.js

```diff
@@ -247,6 +247,7 @@ const criticalCssConfig = (function () {
             'p',
             /\.content-header.*/,
             /\.meta.*/,
+            '.hidden',
```

Comment:
Is this used? Or was `visuallyhidden` meant?

Reply:
It's intended: it's used in the profile login controls.